### DISCUSSION TITLE
Cannot verify my account

### DISCUSSION THREAD

**Discussion by ruvido** [pockethost/pockethost#161] [Q&A]
I am waiting for the confirmation email but it does not come (tried "resend" several times).
Is the sending email service working?

Opening a new account with a different email worked(email arrived). But "resending" for confirmation seems to be down

**Comment by benallfree** [comment]
@ruvido I can check it, but last time this happened someone had input their email address incorrectly. If you'd like further support, please send ben@pockethost.io your info and I'll research it.

**Comment by jgreywolf** [comment]
Is this email still a good email for a similar issue?  I have validated my email address as displayed on the account page, but still not getting the resend confirmation email

**Reply by benallfree** [reply]
Hello, please DM me on Discord @noaxis

**Reply by benallfree** [reply]
Or yes email :)
